### ISSUE TITLE
普通卷积计算和torch存在误差，且堆叠层数加深误差累计，但是group卷积正常

### ISSUE THREAD

**Issue by fxwfzsxyq** [PaddlePaddle/Paddle#56538]
### bug描述
普通卷积计算和torch存在误差，且堆叠层数加深误差累计，但是group卷积正常
# -*- coding: utf-8 -*-

import paddle.nn as pnn
import torch.nn as tnn
import torch
import paddle
torch.set_printoptions(precision=16)
paddle.seed(0)
torch.manual_seed(0)

num = 50
p_list = pnn.LayerList()
t_list = tnn.ModuleList()
for i in range(num):
    # p_list.append(pnn.Conv2D(32,32,3,1,1))
    # t_list.append(tnn.Conv2d(32,32,3,1,1))
    p_list.append(pnn.Conv2D(32, 32, 3, 1, 1,groups=32))
    t_list.append(tnn.Conv2d(32, 32, 3, 1, 1,groups=32))

pmodel = pnn.Sequential(*p_list)
tmodel = tnn.Sequential(*t_list)

print(pmodel.state_dict().keys())
print(tmodel.state_dict().keys())

pp_weights = pmodel.state_dict()
pt_weights = {}
for key in pp_weights.keys():
    torch_key = key.replace('_variance','running_var')
    torch_key = torch_key.replace('_mean','running_mean')
    pt_weights[torch_key] = torch.Tensor(pp_weights[key].numpy())
tmodel.load_state_dict(pt_weights)
pmodel.eval()
tmodel.eval()
print(pmodel)
print(tmodel)

ptimg = torch.rand((1,32,32,32))
ppimg = paddle.to_tensor(ptimg.numpy())
print((ptimg.numpy() - ppimg.detach().numpy()).mean())

pout = pmodel(ppimg)
tout = tmodel(ptimg)

print(abs((pout.numpy() - tout.detach().numpy())).mean())

**Comment by fxwfzsxyq** [comment]
> ### bug描述 Describe the Bug
> # -_- coding: utf-8 -_-
> import paddle.nn as pnn import torch.nn as tnn import torch import paddle torch.set_printoptions(precision=16) paddle.seed(0) torch.manual_seed(0)
> 
> num = 50 p_list = pnn.LayerList() t_list = tnn.ModuleList() for i in range(num): # p_list.append(pnn.Conv2D(32,32,3,1,1)) # t_list.append(tnn.Conv2d(32,32,3,1,1)) p_list.append(pnn.Conv2D(32, 32, 3, 1, 1,groups=32)) t_list.append(tnn.Conv2d(32, 32, 3, 1, 1,groups=32))
> 
> pmodel = pnn.Sequential(*p_list) tmodel = tnn.Sequential(*t_list)
> 
> print(pmodel.state_dict().keys()) print(tmodel.state_dict().keys())
> 
> pp_weights = pmodel.state_dict() pt_weights = {} for key in pp_weights.keys(): torch_key = key.replace('_variance','running_var') torch_key = torch_key.replace('_mean','running_mean') pt_weights[torch_key] = torch.Tensor(pp_weights[key].numpy()) tmodel.load_state_dict(pt_weights) pmodel.eval() tmodel.eval() print(pmodel) print(tmodel)
> 
> ptimg = torch.rand((1,32,32,32)) ppimg = paddle.to_tensor(ptimg.numpy()) print((ptimg.numpy() - ppimg.detach().numpy()).mean())
> 
> pout = pmodel(ppimg) tout = tmodel(ptimg)
> 
> print(abs((pout.numpy() - tout.detach().numpy())).mean())
> 
> ### paddlepaddle：2.5.1
> ### torch: 1.7.1
> ### 其他补充信息 Additional Supplementary Information
> _No response_
